src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -200,6 +202,10 @@ public boolean hasCiteKey() {
      * Returns this entry's type.
      */
     public String getType() {
+        return type.getValue();
+    }
+
+    public StringProperty getTypeProperty() {
```

Comment:
The convention is that such methods are not prefixed with `get`, i.e., use `typeProperty()` instead.

Reply:
Renamed.